types/time.go

```diff
@@ -2108,26 +2108,26 @@ var monthAbbrev = map[string]gotime.Month{
 type dateFormatParser func(t *MysqlTime, date string, ctx map[string]int) (remain string, succ bool)
 
 var dateFormatParserTable = map[string]dateFormatParser{
-	"%b": abbreviatedMonth,           // Abbreviated month name (Jan..Dec)
-	"%c": monthNumeric,               // Month, numeric (0..12)
-	"%d": dayOfMonthNumericTwoDigits, // Day of the month, numeric (00..31)
-	"%e": dayOfMonthNumeric,          // Day of the month, numeric (0..31)
-	"%f": microSeconds,               // Microseconds (000000..999999)
-	"%h": hour24TwoDigits,            // Hour (01..12)
-	"%H": hour24TwoDigits,            // Hour (01..12)
-	"%I": hour24TwoDigits,            // Hour (01..12)
-	"%i": minutesNumeric,             // Minutes, numeric (00..59)
-	"%j": dayOfYearThreeDigits,       // Day of year (001..366)
-	"%k": hour24Numeric,              // Hour (0..23)
-	"%l": hour12Numeric,              // Hour (1..12)
-	"%M": fullNameMonth,              // Month name (January..December)
-	"%m": monthNumericTwoDigits,      // Month, numeric (00..12)
-	"%p": isAMOrPM,                   // AM or PM
-	"%r": time12Hour,                 // Time, 12-hour (hh:mm:ss followed by AM or PM)
-	"%s": secondsNumeric,             // Seconds (00..59)
-	"%S": secondsNumeric,             // Seconds (00..59)
-	"%T": time24Hour,                 // Time, 24-hour (hh:mm:ss)
-	"%Y": yearNumericFourDigits,      // Year, numeric, four digits
+	"%b": abbreviatedMonth,      // Abbreviated month name (Jan..Dec)
+	"%c": monthNumeric,          // Month, numeric (0..12)
```

Comment:
These `0` should be `00`?

Reply:
from the https://dev.mysql.com/doc/refman/5.7/en/date-and-time-functions.html#function_date-format

%c Month, numeric (0..12)